tests/testthat/helper-git.R

```diff
@@ -0,0 +1,13 @@
+build_git_demo <- function(example_dir = "example", db_config = "db_config.yml") {
```

Comment:
Does this need changing to deal with #15 moving to `dettl_config.yml`? Why  is it necessary as an argument though? Should each example not have a configuration inside it?

Reply:
I've updated the name to `dettl_config.yml`. I think we still need this arg for cases when the config is not in the current working directory. So for example in the vignette when the config is in the `inst` directory.

Reply:
I see what's going on here now